### DISCUSSION TITLE
Can you publish `swf`?

### DISCUSSION THREAD

**Discussion by hydroper** [ruffle-rs/ruffle#12837] [Q&A]
It's about 1 year ago it was last published.

**Comment by n0samu** [comment]
Yeah we definitely need to publish a new version when we get a chance

**Comment by hydroper** [comment]
I hope you don't mind; I've published it as [`hydroper_swf`](https://crates.io/crates/hydroper_swf) ([repository](https://github.com/hydroper/rust-swf)) for now as I'm almost about to depend on it for a compiler for handling the `Embed` meta-data.

**Comment by danielhjacobs** [comment]
The crate was just updated: https://crates.io/crates/swf. Hopefully this can be automated in CI soon.
